### PR TITLE
Feat/web app

### PR DESCRIPTION
**Pull request by Elbertlum**
creates basic article list for front end

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -1,8 +1,55 @@
 import React from 'react';
 import { render } from 'react-dom';
-import * as App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import ArticlesApp from './App/Reducers';
```
Comment (dtjv) [inline src/client/index.js]: `./App/Reducers` is exporting reducers. `ArticlesApp` just doesn't seem like a correct name.

File: src/client/App/Reducers/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+// import notes from './notes';
```
Comment (dtjv) [inline src/client/App/Reducers/index.js]: remove commented out code

File: src/client/App/Reducers/articles.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
+const note = (state, action) => {
+  switch (action.type) {
+    case 'DELETE_NOTE':
+      if (state.id !== action.id) {
+        // window.console.log('not matching id');
```
Comment (dtjv) [inline src/client/App/Reducers/articles.js]: remove commented out code

File: src/client/App/Reducers/articles.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    case 'EDIT_TEXT':
+      return state.map(a =>
+        article(a, action)
+      );
+    default:
+      return state;
+  }
+};
+
+export default articles;
```
Comment (dtjv) [inline src/client/App/Reducers/articles.js]: this design exposes only 1 reducer - and that reducer is, for some reason chained to all others. now you'll never be able to load 1 articles or 1 note. (unless I'm missing something). also, a reducer should be scoped to a particular portion of the state, not multiple parts. so a reducer that has a switch on ARTICLE and NOTE is mixing state.

File: src/client/App/Components/Edit.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+
+let textarea = '';
```
Comment (dtjv) [inline src/client/App/Components/Edit.js]: no globals

File: src/client/App/Components/Edit.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+const handleText = (event) => {
+  textarea = event.target.value;
+  // window.console.log(textarea);
+};
+
+const Edit = ({ note, onCancel, id, editText }) => (
```
Comment (dtjv) [inline src/client/App/Components/Edit.js]: you've got state here. i think this component will have to change.

File: src/client/App/Actions/index.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+export const goTo = (url) => {
+  window.location = url;
+};
+
+export const loadArticle = (article) => (
```
Comment (dtjv) [inline src/client/App/Actions/index.js]: i can't find where `loadArticle` is ever called.

File: src/client/App/Reducers/index.js (hunk 2 above)
Comment (Elbertlum) [inline src/client/App/Reducers/index.js]: Refactor uses this line, uncommented

File: src/client/index.js (hunk 1 above)
Comment (Elbertlum) [inline src/client/index.js]: Changed to Reducers

File: src/client/App/Reducers/articles.js (hunk 3 above)
Comment (Elbertlum) [inline src/client/App/Reducers/articles.js]: removed

File: src/client/App/Reducers/articles.js (hunk 4 above)
Comment (Elbertlum) [inline src/client/App/Reducers/articles.js]: changed to two reducers, notes and articles

**Pull request merged**